python/kwiver/arrows/serialize/json/serialize_activity.cxx

```diff
@@ -46,3 +49,4 @@ void serialize_activity(py::module &m)
                           kwiver::vital::activity,
                           kwiver::arrows::serialize::json::activity > );
 }
+}}}
```

Comment:
Minor nit, based on the namespace example in [style guide](https://github.com/Kitware/kwiver/blob/master/doc/code-style.rst#ws-whitespace), the parentheses should be in separate lines